CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Use `Logger.debug` to print warning about closing keep-alive connections.
 
+## Fixed
+
+- Properly close lingering workers. Send `{:stop, :normal, ...}` when RAxx.response :body is binary 
```

Comment:
capitalisation. `Raxx.Response`

Reply:
I'm not clear on this?

Reply:
I'm saying `RAxx.response` -> `Raxx.Response`. that is the struct that has a body

Reply:
ok